modules/ROOT/pages/app-dashboards.adoc

```diff
@@ -0,0 +1,105 @@
+= Builtin Application Dashboard
```

Comment:
Merriam-Webster uses "Built-in" as the adjective, though I realize Chicago Manual of Style recommends closing up in some cases. The former is more common, I think. https://www.merriam-webster.com/dictionary/built-in

Reply:
Not sure what style convention we're supposed to follow though... or what we're using (most) elsewhere in the doc set, though the rest of the page uses "built-in" in various places.